css/fonts.css

```diff
@@ -0,0 +1,43 @@
+@font-face {
+  font-family: "Montserrat";
+  src: 
+  url("../fonts/MontserratAlternates-Bold.eot"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5516260)